### PR TITLE
Allow all dependabot pull requests

### PR DESCRIPTION
**Pull request by perlpunk**
Issue: https://progress.opensuse.org/issues/157462

Allowing only security updates has the disadantage that there might be security update conflicts which require non security updates, and doing them manually is more work.

Let's try without any limit first.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ updates:
     allow:
       # Allow both direct and indirect updates for all packages
       - dependency-type: all
-    open-pull-requests-limit: 0
+    # open-pull-requests-limit: 0
```
Comment (kalikiana) [inline .github/dependabot.yml:12]: How is this related to security updates?

And in general please remove the line if you intend to remove it.

Comment (perlpunk) [reply .github/dependabot.yml:12]: open-pull-requests-limit limits the numer of dependabot pull requests for general updates.
It does not apply to security updates, so regardless of the limit dependabot will create security PRs.
Setting the limit to 0 is an undocumented workaround, because there is no other way to tell dependabot to only do securoty updates.

**Review by Martchus — APPROVED** [review]

**Review by okurz — COMMENTED** [review]
Just remove the line as @kalikiana asked

File: .github/dependabot.yml (hunk 1 above)
Comment (perlpunk) [reply .github/dependabot.yml]: removed the line

**Comment by perlpunk** [comment]
Fullstack failing:
https://app.circleci.com/pipelines/github/os-autoinst/openQA/13196/workflows/ddb935c7-31dd-4beb-877c-25ef1e703b4d/jobs/123231
```
[14:03:42] t/33-developer_mode.t .. 17/?     # Unexpected Javascript console errors, waiting for connection opened: [
    #   {
    #     level     => "SEVERE",
    #     message   => "http://localhost:9526/asset/3906633cf0/ws_console.js 8 WebSocket connection to 'ws://localhost:9528/liveviewhandler/tests/1/developer/ws-proxy' failed: Error during WebSocket handshake: Unexpected response code: 302",
    #     source    => "network",
    #     timestamp => 1710857067816,
    #   },
    # ]

    #   Failed test 'No unexpected js warnings'
    #   at /home/squamata/project/t/lib/OpenQA/Test/FullstackUtils.pm line 123.
    # Looks like you failed 1 test of 9.
[14:03:42] t/33-developer_mode.t .. 20/? 
```
I will create a ticket

**Comment by perlpunk** [comment]
And another failure
https://app.circleci.com/pipelines/github/os-autoinst/openQA/13196/workflows/ddb935c7-31dd-4beb-877c-25ef1e703b4d/jobs/123228
```
[14:08:28] t/ui/23-audit-log.t ........................ 12/? 
    #   Failed test 'most rows filtered out when searching for table create events'
    #   at t/ui/23-audit-log.t line 40.
    #          got: '8'
    #     expected: '3'
    # Looks like you failed 1 test of 22.
[14:08:28] t/ui/23-audit-log.t ........................ 13/? 
#   Failed test 'clickable events'
#   at t/ui/23-audit-log.t line 152.
[14:08:28] t/ui/23-audit-log.t ........................ 14/? # Looks like you failed 1 test of 14.
[14:08:28] t/ui/23-audit-log.t ........................ Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/14 subtests 
```

**Review by okurz — APPROVED** [review]
